### PR TITLE
Fixes #1367 try get value

### LINKED ISSUE #1367: Add a method to evaluate the value of the aprameter and return false if it's not possible TryGetValue

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

**Comment by msevestre** [comment]
@Yuri05 This is to ensure that we do not catch all the time in debug mode in PKSim to speed up the debug process

File: src/OSPSuite.Core/Domain/Quantity.cs [hunk 1]
```diff
@@ -38,6 +40,12 @@ public interface IQuantity : IFormulaUsable, IUsingFormula, IWithValueOrigin
       /// </summary>
       double ValueInDisplayUnit { get; set; }
 
+      /// <summary>
+      ///    The value in the displayed unit
+      /// </summary>
+      bool TryGetValueInDisplayUnit(out double result);
```
Comment (Yuri05) [inline src/OSPSuite.Core/Domain/Quantity.cs:46]: Should `TryGetValueInDisplayUnit `be used **only** for tests? Then: why not defining it as Extension Method?

Comment (msevestre) [reply src/OSPSuite.Core/Domain/Quantity.cs:46]: No not for test. I want to use it in real code when the debugger is running, catching the error all the time creates a lot of overhead. It makes working with the tool while debugging very hard (initial concentration formula are all invalid and the whole UI freezes). This is a problem in debug mode only that I wanted to fix for a while. Now the pain is too big so it's required

Comment (msevestre) [reply src/OSPSuite.Core/Domain/Quantity.cs:46]: But we can define it as extension method. I don't see a problem

Comment (msevestre) [reply src/OSPSuite.Core/Domain/Quantity.cs:46]: This will be used in ParameterDTO in pksim

Comment (Yuri05) [reply src/OSPSuite.Core/Domain/Quantity.cs:46]: ok, if not for tests onle, then it's OK I think

**Review by Yuri05 — APPROVED** [review]

**Comment by msevestre** [comment]
Well it did not improve performance so I'll revert this and implement a fix in pksim directly

**Pull request merged**